Gruntfile.coffee

```diff
@@ -892,6 +892,9 @@ module.exports = (grunt) ->
 						"The “contentinfo” role is unnecessary for element “footer”."
 						"The “navigation” role is unnecessary for element “nav”."
 						"The “banner” role is unnecessary for element “header”."
+						"Element “search” not allowed as child of element “main” in this context. (Suppressing further errors from this subtree.)"
+						"Element “search” not allowed as child of element “section” in this context. (Suppressing further errors from this subtree.)"
+						"Element “search” not allowed as child of element “div” in this context. (Suppressing further errors from this subtree.)"
```

Comment:
FWIW the HTML validator officially added support for the ``<search>`` element in early June via validator/validator#1828, but... but there haven't been any "proper" new releases of the validator since then ([latest tag](https://github.com/validator/validator/tags) is called ``latest``... not ideal since it's at risk of changing constantly).

If a new version number tag gets rolled-out at some point, hopefully ``grunt-html`` will quickly receive a new release as well ([its latest release](https://github.com/validator/validator/tags) came out right after the validator's latest numbered tag).

Maybe it'd be possible to override WET's ``grunt-html`` dependency to use a newer validator commit hash in the meantime...